google-ads-examples/src/main/java/com/google/ads/googleads/examples/targeting/AddCustomerNegativeCriteria.java

```diff
@@ -0,0 +1,138 @@
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new AddCustomerNegativeCriteria().runExample(googleAdsClient, params.customerId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Creates customer negative criteria.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  public static void runExample(GoogleAdsClient googleAdsClient, long customerId) {
+
+    try (CustomerNegativeCriterionServiceClient customerNegativeCriterionServiceClient =
+        googleAdsClient.getLatestVersion().createCustomerNegativeCriterionServiceClient()) {
+
+      // Creates a negative customer criterion excluding the content label type of 'TRAGEDY'.
+      CustomerNegativeCriterion tragedyCriterion =
+          CustomerNegativeCriterion.newBuilder()
+              .setContentLabel(
+                  ContentLabelInfo.newBuilder().setType(ContentLabelType.TRAGEDY).build())
+              .build();
+      // Creates a negative customer criterion excluding the placement with url
+      // 'http://www.example.com'.
+      CustomerNegativeCriterion placementCriterion =
+          CustomerNegativeCriterion.newBuilder()
+              .setPlacement(
+                  PlacementInfo.newBuilder()
+                      .setUrl(StringValue.of("http://www.example.com"))
+                      .build())
+              .build();
+
+      // Creates the operations.
+      CustomerNegativeCriterionOperation tragedyCriterionOperation =
+          CustomerNegativeCriterionOperation.newBuilder().setCreate(tragedyCriterion).build();
+      CustomerNegativeCriterionOperation placementCriterionOperation =
+          CustomerNegativeCriterionOperation.newBuilder().setCreate(placementCriterion).build();
+      List<CustomerNegativeCriterionOperation> operations =
+          new ArrayList<>(Arrays.asList(tragedyCriterionOperation, placementCriterionOperation));
```

Comment:
Don't need new ArrayList() and Arrays.asList(), both of these return List - just use Arrays.asList()